beetsplug/convert.py

```diff
@@ -531,6 +522,21 @@ def convert_on_import(self, lib, item):
                 )
                 util.remove(source_path, False)
 
+    def _get_art_resize(self, artpath):
```

Comment:
It might be nice to write a docstring for this new method to describe what it does?